articles/quickstart/native/ionic4/01-login.md

```diff
@@ -0,0 +1,405 @@
+---
+title: Login
+default: true
+description: This tutorial demonstrates how to add user login to an Ionic 4 application using Auth0.
+budicon: 448
+topics:
+  - quickstarts
+  - native
+  - ionic4
+github:
+  path: 01-Login
+contentType: tutorial
+useCase: quickstart
+---
+
+<%= include('../_includes/_getting_started', { library: 'Ionic' }) %>
+
+<%= include('../../../_includes/_callback_url') %>
+
+The **Callback URL** to be used for your application includes your app's package ID which is found in the `config.xml` file for your app.
```

Comment:
If the customer hasn’t added iOS or Android as a platform, they won’t have a `config.xml` file yet.